spec/game_spec.rb

```diff
@@ -0,0 +1,103 @@
+require './lib/classes.rb'
+  	
+RSpec.describe Check_win do
+	describe "#check win" do
+		win= Check_win.new
```

Comment:
There should be space around operator; 
`win= Check_win.new` ------Bad
`win = Check_win.new` ------Good
One of the Microverse guideline state; students should make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide

_Kindly apply this to the rest the operator in your code._

Reply:
perfect

Reply:
specs are failing if i break some code in tested class? this part I don't get it

Reply:
😁, am simply saying your code test is working fine.